scripts/upload-images

```diff
@@ -22,71 +22,3 @@ set -ex
 # even though it is not necessary. This script can be deleted completely when
 # our github hook code is deprecated.
 exit 0
```

Comment:
This script has been exiting here for a while, so just cleaning up the rest of the (unused) portion of it.